include/mrdocs/Metadata/Javadoc.hpp

```diff
@@ -81,14 +85,27 @@ enum class Admonish
 
 /** Parameter pass direction.
 */
-enum class ParamDirection : int
+enum class ParamDirection
 {
     none,
     in,
     out,
     inout
 };
 
+/** Which parts of the documentation to copy.
```

Comment:
The part to copy comes from the command (@copydoc, @copybrief, @copydetails). This is only an internal detail, right? Should it be in a .hpp file?

Reply:
@alandefreitas We don't have a way to hide implementation detail for `Info` and `doc::Node` derived types right now... they are directly used by generators